test/unittests/test_dd_synthesis.cpp

```diff
@@ -62,7 +62,7 @@ TEST_P(TestDDSynth, GenericDDSynthesisTest) {
     EXPECT_TRUE(ttDD.p != nullptr);
 
     DDSynthesizer synthesizer(tt.nInputs());
-    const auto&   qc   = synthesizer.synthesize(ttDD, dd);
+    const auto&   qc   = synthesizer.synthesize(ttDD, {}, 0U, tt.nOutputs(), dd);
```

Comment:
as stated in a comment above, this change should not be necessary as there should be a general purpose function for that.